readthedocs/search/utils.py

```diff
@@ -67,3 +68,25 @@ def _get_document(model, document_class):
     for document in documents:
         if str(document) == document_class:
             return document
+
+
+def _indexing_helper(html_objs, wipe=False):
+    """
+    Helper function for reindexing and wiping indexes of projects and versions.
+
+    If ``wipe`` is set to False, html_objs are deleted from the ES index,
+    else, html_objs are indexed.
+    """
+    from readthedocs.search.tasks import index_objects_to_es, delete_objects_in_es
```

Comment:
This should check if `html_objs` exists -- it might not.

Reply:
I have added the checks to prevent this.